.circleci/config.yml

```
@@ -24,7 +24,7 @@ jobs:
             source dev_env.sh
             # BUG https://jira.talendforge.org/browse/TDL-15395
             echo "pylint is skipping the following: $PYLINT_DISABLE_LIST"
-            pylint tap_jira -d "$PYLINT_DISABLE_LIST,unsupported-assignment-operation,unsupported-membership-test,unsubscriptable-object,dangerous-default-value,too-many-instance-attributes"
+            pylint tap_jira -d "$PYLINT_DISABLE_LIST,unsupported-assignment-operation,unsupported-membership-test,unsubscriptable-object,dangerous-default-value,too-many-instance-attributes,too-many-function-args"
```

```suggestion
            pylint tap_jira -d "$PYLINT_DISABLE_LIST,unsupported-assignment-operation,unsupported-membership-test,unsubscriptable-object,dangerous-default-value,too-many-instance-attributes"
```

This failed pylint because it caught a real error:
```sh
tap_jira/streams.py:195:19: E1121: Too many positional arguments for classmethod call (too-many-function-args)
```

`tap_jira/streams.py:195:19` pointed to [this line in this commit ](https://github.com/singer-io/tap-jira/pull/73/commits/76593dcf65058b6233cbe3388dab01428890128e#diff-d8c84b98b7e944497c25b6b8768bb33fbab0dd4d091a856c9b96f3250ccddec0R195)
```python
response = Context.retrieve_timezone(page_num_offset)
```